openapi/src/main/java/io/helidon/openapi/OpenAPISupport.java

```diff
@@ -93,8 +94,11 @@
  * the server uses none of these builder methods and does not provide a static
  * {@code openapi} file, then the {@code /openapi} endpoint responds with a
  * nearly-empty OpenAPI document.
+ *
+ * @param <T> concrete subclass which extends OpenAPISupport
+ * @param <B> concrete builder subclass which extends OpenAPISupport.Builder
  */
-public class OpenAPISupport implements Service {
+public abstract class OpenAPISupport<T extends OpenAPISupport<T, B>, B extends OpenAPISupport.Builder<T, B>> implements Service {
```

Comment:
The type `B` is used as the return type of `Builder#me()`. Otherwise there is no way for the builder methods that are implemented on the abstract class to return `this` correctly-typed to the builder subclass. Or at least I've not seen one. If there is I would love to do away with `me()`.